web/pdf_page_view.js

```diff
@@ -1085,7 +1077,8 @@ class PDFPageView {
     return result;
   }
 
-  paintOnSvg(wrapper) {
+  paintOnSvg() {
+    const wrapper = this.#canvasWrapper;
```

Comment:
This needs to be moved *below* the `PDFJSDev` checks, since otherwise it's not getting excluded in Firefox.